python/ray/dashboard/modules/metrics/dashboards/train_grafana_dashboard_base.json

```diff
@@ -85,11 +85,11 @@
         "name": "TrainRunName",
         "options": [],
         "query": {
-          "query": "label_values(ray_train_report_total_blocked_time_s{{{global_filters}}}, ray_train_run_name)",
+          "query": "query_result(count by (ray_train_run_name)(last_over_time(ray_train_report_total_blocked_time_s{{SessionName=~\"$SessionName\",{global_filters}}}[$__range])))",
```

Comment:
I realized that `ray_train_report_total_blocked_time_s` may not be recorded in the case that the user does not call `ray.train.report`. Would it be better to use a metric that will always be recorded? E.g. right now `ray_train_worker_group_start_total_time_s` should always be there.